src/Adapter/Product/Combination/Validate/CombinationValidator.php

```diff
@@ -90,6 +90,8 @@ private function validateStock(Combination $combination): void
         $this->validateCombinationProperty($combination, 'low_stock_threshold', ProductConstraintException::INVALID_LOW_STOCK_THRESHOLD);
         $this->validateCombinationProperty($combination, 'low_stock_alert', ProductConstraintException::INVALID_LOW_STOCK_ALERT);
         $this->validateCombinationProperty($combination, 'available_date', ProductConstraintException::INVALID_AVAILABLE_DATE);
+        $this->validateCombinationLocalizedProperty($combination, 'available_later', ProductConstraintException::INVALID_AVAILABLE_LATER);
+        $this->validateCombinationLocalizedProperty($combination, 'available_now', ProductConstraintException::INVALID_AVAILABLE_NOW);
```

Comment:
All good here. Just I now see we forgot to add constraints to product available_now and available_later fields in [s](https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Form/Admin/Sell/Product/Stock/AvailabilityType.php#L88)
ping @jolelievre :smile:

Reply:
https://github.com/PrestaShop/PrestaShop/issues/29454
created related issue